syncGraph.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python3
+
+'''
+' @file   syncGraph.py
+' @desc   Grabs input JSON file of posts and edges, inserts into PostRelatedness ...
+'         ... the edges and inserts into PostCentrality the scores from posts.
+' @notes  The file has scrape_id's. For inserting into our DB, we need to search for our id's (seen below in queries).
+'''
+
+### Imports
+
+from datetime import datetime
+from collections import defaultdict
+import mysql.connector
+import json
+import sys
+import os
+
+
+### Constants
+
+MEWS_CONFIG_FILEPATH = 'config/inter-mews.json'
+VERBOSE = True
+
+
+### Functions
+
+def vprint(s):
+    '''
+    @desc   Prints on VERBOSE mode
+    --
+    @param  s  String to print
+    '''
+    global VERBOSE
+    if VERBOSE: print(s)
+
+
+def loadConfig(filepath):
+    '''
+    @desc   Loads the mysql config json files
+    --
+    @param  filepath  path to config file
+    '''
+    with open(filepath) as f:
+        return json.load(f)
+
+
+def connectSQL(config):
+    '''
+    @desc   Connects to mysql
+    --
+    @param  config  mysql.connector config object
+    '''
+    try:
+        return mysql.connector.connect(**config)
+    except mysql.connector.Error:
+        raise
+        return None
+
+
+def load_json(fpath):
+    '''
+    @desc    parses JSON graph file
+    --
+    @param   fpath    file path for json file
+    @return  posts    dict with post data (centrality, ocr, rel_txt)
+    @return  edges    dict with edge data
+    '''
+
+    vprint(f'Loading in graph from "{fpath}"')
+
+    # Grab JSON
+    with open(fpath) as f:
+        data = json.load(f)
+
+    posts = data['posts']
+    edges = data['edges']
+
+    return posts, edges
+
+
+def getSubimageWeightsMeta(edges, source, target):
+    '''
+    @desc  Returns subimage weight (float, else None) and metadata (string, else None)
+    '''
+    sub_edges = edges[source][target].get('subimage', [])
+    labels = []
+    sw = 0 if len(sub_edges) > 0 else None
```

Comment:
I might suggest more descriptive variable names ('weight' and 'metadata', maybe?)

Reply:
Note taken. Will update.